File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session1/p9.c
#include<stdio.h>
void main(){

	int row;
	printf("Enter num of row:\n");
	scanf("%d",&row);
	char ch='A';
	for(int i=1;i<=row;i++){
		for(int j=1;j<=row;j++){
			printf("%c ",ch);
		}
		ch++;
		printf("\n");
	}
}

File: flutter-main/flutter-main/Pattern2/p9.dart
import 'dart:io';
void main () {
        int num=4;
        int row=num;
        for (int i=1; i<=num; i++, row--) {
        	int dec=i;
                for(int j=1; j<=row; j++, dec++) {
                        stdout.write ("${dec}  ");

                }
                print("");
        }
}

